locals.tf

```diff
@@ -78,7 +78,10 @@ locals {
       ]
       # The extra containers of the repo_server pod must have resource requests/limits in order to allow this component
```

Comment:
To avoid repetition across all resources, for example we can define the following on locals:
 ```
 null_values = function(map) {
    { for k, v in map : k => v if v != null }
  }
```
  and 
```
  requests = local.null_values(var.resources.repo_server.requests)
limits   = local.null_values(var.resources.repo_server.limits)

```
what do you think ? Please note, this should be tested first :)

Reply:
I did not know we could create functions in Terraform :open_mouth:  Are you sure we can? I did not find anything in the documentation...

Reply:
Sorry my bad  I was confending with map function no longer available
please ignore this message ;)

Reply:
Ok! Thank you, @omohammed95 !